demo-project/lightsail.json

```diff
@@ -0,0 +1,15 @@
+{
+  "serviceName": "kedro-viz-live-demo",
+  "containers": {
+    "kedro-viz-live-demo": {
+      "image": "public.ecr.aws/g0x0s3o2/kedro-viz-live-demo:4.2.0",
```

Comment:
Does `4.2.0` need to change when we release a new version of Viz? That's our current version isn't it?

Reply:
oh yes such a good point. I forgot about this. Let me fix it.

Reply:
indeed great point - that reminds me that our `make version` script needs to update the `.version` file with the latest version as well.

Reply:
@studioswong that actually was done in this PR. I just forgot about this JSON

Reply:
indeed just saw the update the `versioning.py` - I'm blind 🙈 

I tried running the versioning script and it's returning an error due to it referring to the old demo project name - see below comment.